classes/options-general.php

```diff
@@ -445,7 +445,7 @@ function print_page() {
 																                                           value=""/>&nbsp;<a
 																	class="button" <?php disabled( $disabled_api ); ?>
 																	onclick="codeAddress();return false;"
-																	href="#"><?php _e( 'Geocode Address', 'simplemap' ); ?></a>
+																	href="#"><?php _e( 'Geocode Address', 'SimpleMap' ); ?></a>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 127 characters

<!-- Comment by [SideCI](https://sideci.com) -->